cmd/webui/assets/index.html

```diff
@@ -95,15 +95,18 @@
         <h1>ACH File Parser</h3>
             <p>
-                This tool converts ACH files into their JSON definition using Moov's <a href="https://github.com/moov-io/ach">ACH library</a>. Paste an ACH file on the left and the equivalent JSON will be generated to the right.
+                This tool converts ACH files into their JSON definition or a human-readable format using Moov's <a href="https://github.com/moov-io/ach">ACH library</a>. Paste an ACH file on the left and the formatted version will be generated to the right.
+            </p>
+            <p>
                 For an example, try some of our <a href="https://github.com/moov-io/ach/tree/master/test/testdata">test files</a>.
             </p>
     </header>
-    <textarea id="jsoninput" name="jsoninput" cols="80" rows="40" placeholder="Paste your ACH file contents here..." required></textarea>
-    <textarea id="jsonoutput" name="jsonoutput" cols="80" rows="40" readonly></textarea>
+    <textarea id="contentinput" name="contentinput" cols="80" rows="40" placeholder="Paste your ACH file contents here..." required></textarea>
+    <textarea id="contentoutput" name="contentoutput" cols="80" rows="40" readonly></textarea>
     <div id="buttons">
-        <button type="submit" onclick="convertToJson(jsoninput.value)">Parse ACH file</button>
-        <button type="button" onclick="convertToAch(jsoninput.value)">Parse JSON contents</button>
+        <button type="submit" onclick="convertToJson(contentinput.value)">Parse ACH file to JSON</button>
+        <button type="button" onclick="convertToReadable(contentinput.value)">Parse ACH file to human-readable format</button>
+        <button type="button" onclick="convertToAch(contentinput.value)">Parse JSON to ACH file</button>
```

Comment:
Maybe the following text is better?

- View as JSON
- Human Readable
- View as Nacha

Reply:
I like that better.

Reply:
I like shortening the text but could be ambiguous to unfamiliar users what the input should be, thoughts on?
- ACH to JSON
- ACH to human-readable
- JSON to ACH

Reply:
I had a similar thought. Going to try a few options and see which is the easiest to understand.